src/dapps/nft.ts

```diff
@@ -189,7 +196,13 @@ export namespace NFT {
                 type: 'boolean'
               }
             },
-            required: ['bodyShapes', 'category', 'description', 'rarity'],
+            required: [
+              'bodyShapes',
+              'category',
+              'description',
+              'rarity',
+              'isSmart'
```

Comment:
Is it ok to add `isSmart` here?

Reply:
yes, it was missing, but it should be required